test/integration/visitor_not_authorized_to_view_user_data_test.rb

```diff
@@ -0,0 +1,14 @@
+require 'test_helper'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.